liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PluginConfigSupport.java

```diff
@@ -214,7 +214,7 @@ protected boolean isAppConfigInSourceServerXml() {
     
     protected String getAppsDirectory() {    
     
-        if (appsDirectory != null && !appsDirectory.isEmpty())
+        if (appsDirectory != null && (appsDirectory.equals("apps") || appsDirectory.equals("dropins")))
```

Comment:
Expected to see a warning message that the appsDirectory entry is not valid if it is set and is not `apps` or `dropins`.